api/src/components/coauthor/controller.ts

```diff
@@ -43,6 +43,19 @@ export const updateAll = async (
             });
         }
 
+        // Check if duplicate coAuthors in array
+        const valueArr = event.body.map(function (coAuthor) {
+            return coAuthor.email;
+        });
```

Comment:
line break between block if poss & (minor) same as above: arrow function could be passed to .some() could be more readable